core/valuer.go

```diff
@@ -85,9 +85,9 @@ func toValue(value any, tags reflect.StructTag) Value {
 		return NewText()
 	}
 	uv := reflectx.Underlying(rv)
-	if reflectx.IsNil(uv) {
-		return NewText()
-	}
+	// if reflectx.IsNil(uv) { // no: handle null properly in relevant widgets; essential for list / table
```

Comment:
@rcoreilly usually i do not recommend to leave such comments, on my opinion it adds more cognitive complexity. if it is important it is ok to keep related info in doc comments before function signature.

just my 2 cents

Reply:
code itself could be removed, if someone is interested in then it is always possible to take a look at git history.

Reply:
Yes good point -- this was here for @kkoreilly's benefit in reviewing but I would probably forget to remove the comment and now that I've made this PR with all the discussion, clearly this is the proper location for such comments!